frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -611,7 +621,7 @@ function getBoundingBoxMenuOptions({
     const hideBoundingBoxMenuItem: MenuItemType = {
       key: "hide-bounding-box",
       onClick: () => {
-        hideBoundingBox(clickedBoundingBoxId);
+        Actions.hideBoundingBox(Store.dispatch, clickedBoundingBoxId);
```

Comment:
while it's not super elegant (since it doesn't feel 100% consistent), it is kind of deliberate. After I converted mapDispatchToProps to the Actions helper, I removed several methods from it because I felt that they only added clutter and no benefit (most of them were in the form of `action(a, b, c) => dispatch(action(a, b, c))`). However, `hideBoundingBox` at least adds some value because it internally uses `changeUserBoundingBoxAction` (which would be more verbose for the caller when changing the visibility).

I think, what would be even better is to put the `hideBoundingBox` code into an own action creator in the corresponding module. However, we always suffix our action creators with "action" and then the function also returns such an action (e.g., `doStuffAction` returns `DoStuffAction`). If we add a `hideBoundingBoxAction` function, it would still return a `ChangeUserBoundingBoxAction` which is a bit weird..
I'm not really happy with the code, but felt like I shouldn't digress further in this PR..